addons/base_automation/tests/test_mail_composer.py

```diff
@@ -34,11 +34,12 @@ def test_mail_composer_test_tour(self):
             'model_id': self.env.ref('mail.model_mail_compose_message').id,
         })
         automation.write({'action_server_ids': [(4, server_action.id)]})
-
+        user = self.env["res.users"].create({"name": "Not A Demo User", "login": "nadu"})
```

Comment:
But do we need a user ? I mean, purpose is to test recipients, so it seems more logical to use partners ?